core/templates/dev/head/galleries/Gallery.js

```diff
@@ -95,16 +95,32 @@ oppia.controller('Gallery', [
     });
 
     $scope.allExplorationsInOrder = [];
+    $scope.allCollectionsInOrder = [];
 
     // Called when the page loads, and after every search query.
     var _refreshGalleryData = function(data, hasPageFinishedLoading) {
       $scope.searchIsLoading = false;
       $scope.allExplorationsInOrder = data.explorations_list;
+      $scope.allCollectionsInOrder = data.collections_list;
       $scope.finishedLoadingPage = hasPageFinishedLoading;
       $rootScope.loadingMessage = '';
     };
 
     $scope.pageLoaderIsBusy = false;
+    $scope.showMoreExplorations = function() {
```

Comment:
This is a mistake during merging. I wasn't aware that this function has been moved to `activityTilesInfinityGrid.js`